lib/v2/japanpost/radar.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    'japanpost.jp': {
+        _name: '日本郵便',
+        'trackings.post': [
+            {
+                title: '郵便・荷物の追跡',
+                docs: 'https://docs.rsshub.app/routes/other#ri-ben-you-bian-you-bian-zhui-ji-サービス',
+                source: '/services/srv/search/direct',
+                target: (params, url) => {
+                    const reqCode = new URL(url).searchParams.get('reqCodeNo1').toUpperCase();
+                    const locale = new URL(url).searchParams.get('locale').toLowerCase();
+                    if ((reqCode.search(/^(?:\d{11,12}|[A-Z]{2}\d{9}[A-Z]{2})$/) === 0 && locale === 'ja') || locale === 'en') {
+                        return `/japanpost/track/${reqCode}/${locale}`;
+                    }
+                },
```

Comment:
Please remove `rsshub=1` in the docs.

Reply:
You might refer to `rssbud="1"` in the docs?
I've tested on my device, and RSSBud functions normally with this radar rule.